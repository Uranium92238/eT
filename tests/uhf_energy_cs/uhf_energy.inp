system
   name: Cs
   charge: 0
   multiplicity: 2
end system

do
   ground state
end do

method
   uhf
end method

memory
   available: 8
end memory

solver scf
  algorithm:          scf-diis
  energy threshold:   1.0d-11
  gradient threshold: 1.0d-11
end solver scf

geometry
basis: 3-21G
   Cs         0.00000        0.00000        0.00000
end geometry

