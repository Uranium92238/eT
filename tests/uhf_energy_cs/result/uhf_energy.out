


                     eT 1.1 - an electronic structure program

  ------------------------------------------------------------------------
   Author list in alphabetical order:
  ------------------------------------------------------------------------
   J. H. Andersen, A. Balbi, S. Coriani, S. D. Folkestad, T. Giovannini, 
   L. Goletto, T. S. Haugland, A. Hutcheson, I-M. Høyvik, E. F. Kjønstad, 
   H. Koch, T. Moitra, R. H. Myhre, A. C. Paul, S. Roet, M. Scavino, 
   A. Skeidsvoll, Å. H. Tveten
  ------------------------------------------------------------------------
   J. Chem. Phys. 152, 184103 (2020); https://doi.org/10.1063/5.0004713


   This is eT version 1.1.0, Bastion
  ------------------------------------------------------------
  Compiled by:        alexancp
  Compiled on:        NTNU16748
  Configuration date: 2021-01-27 16:09:07 UTC +01:00
  Git branch:         extend-multiplicity-table
  Git hash:           cc9d4c5a20ecb10953c3257ee5d32bd5bc332875
  Fortran compiler:   GNU 9.3.0
  C compiler:         GNU 9.3.0
  C++ compiler:       GNU 9.3.0
  LAPACK type:        MKL
  BLAS type:          MKL
  64-bit integers:    ON
  OpenMP:             ON
  PCM:                OFF
  Forced batching:    OFF
  Runtime checks:     OFF
  ------------------------------------------------------------

  Calculation started: 2021-01-27 19:44:39 UTC +01:00


  :: Input file
  ================

     Note: geometry section is excluded from this print

     system
        name: Cs
        charge: 0
        multiplicity: 2
     end system

     do
        ground state
     end do

     method
        uhf
     end method

     memory
        available: 8
     end memory

     solver scf
       algorithm:          scf-diis
       energy threshold:   1.0d-11
       gradient threshold: 1.0d-11
     end solver scf


  Running on 2 OMP threads
  Memory available for calculation: 8.000000 GB


  :: UHF wavefunction
  ======================

     ==============================================================================
                                     Geometry (angstrom)
     ==============================================================================
        Atom           X                  Y                  Z         # in input
     ==============================================================================
        Basis: 3-21g
        1 Cs     0.000000000000     0.000000000000     0.000000000000        1
     ==============================================================================

     ==============================================================================
                                     Geometry (a.u.)
     ==============================================================================
        Atom           X                  Y                  Z         # in input
     ==============================================================================
        Basis: 3-21g
        1 Cs     0.000000000000     0.000000000000     0.000000000000        1
     ==============================================================================

  - Cholesky decomposition of AO overlap to get linearly independent AOs:

     Linear dependence threshold:             0.10E-05
     Number of atomic orbitals:               37
     Number of orthonormal atomic orbitals:   37

  - Molecular orbital details:

     Number of alpha electrons:              28
     Number of beta electrons:               27
     Number of virtual alpha orbitals:        9
     Number of virtual beta orbitals:        10
     Number of molecular orbitals:           37


  :: Hartree-Fock engine
  =========================

  Drives the calculation of the Hartree-Fock state.

  This is a UHF ground state calculation.
  The following tasks will be performed:

     1) Generate initial SAD density
     2) Calculation of reference state (SCF-DIIS algorithm)


  1) Generate initial SAD density


  2) Calculation of reference state (SCF-DIIS algorithm)

   - Self-consistent field DIIS Hartree-Fock solver
  ----------------------------------------------------

  A DIIS-accelerated Roothan-Hall self-consistent field solver. A least-square 
  DIIS fit is performed on the previous Fock matrices and associated gradients. 
  Following the Roothan-Hall update of the density, the DIIS-fitted Fock 
  matrix is used to get the next orbital coefficients.

  - Hartree-Fock solver settings:

     DIIS dimension:                         8
     Cumulative Fock threshold:       0.10E+01

     Residual threshold:            0.1000E-10
     Energy threshold:              0.1000E-10

     Coulomb screening threshold:   0.1000E-16
     Exchange screening threshold:  0.1000E-14
     Integral cutoff:               0.1000E-16

  - Setting initial AO density to SAD

     Energy of initial guess:             -7530.568190922066
     Number of electrons in guess:           55.000000000000

  Iteration       Energy (a.u.)      Max(grad.)    Delta E (a.u.)
  ---------------------------------------------------------------
     1         -7530.542527047885     0.1562E-02     0.7531E+04
     2         -7530.543242039053     0.6254E-03     0.7150E-03
     3         -7530.543345749134     0.2859E-03     0.1037E-03
     4         -7530.543363707362     0.1677E-04     0.1796E-04
     5         -7530.543363803160     0.3497E-05     0.9580E-07
     6         -7530.543363806697     0.1160E-05     0.3537E-08
     7         -7530.543363807137     0.8127E-07     0.4402E-09
     8         -7530.543363807137     0.3141E-08     0.0000E+00
     9         -7530.543363807134     0.4896E-10     0.3638E-11
    10         -7530.543363807136     0.1118E-10     0.2728E-11
    11         -7530.543363807137     0.4096E-12     0.9095E-12
  ---------------------------------------------------------------
  Convergence criterion met in 11 iterations!

  - Summary of UHF wavefunction energetics (a.u.):

     HOMO-LUMO gap (alpha):          0.289543124715
     HOMO-LUMO gap (beta):           0.696523627790
     Nuclear repulsion energy:       0.000000000000
     Electronic energy:          -7530.543363807137
     Total energy:               -7530.543363807137

  - UHF wavefunction spin expectation values:

     Sz:                   0.50000000
     Sz(Sz + 1):           0.75000000
     S^2:                  0.75022793
     Spin contamination:   0.00022793

  - Timings for the UHF ground state calculation

     Total wall time (sec):              6.74965
     Total cpu time (sec):              13.35199

     Peak memory usage during the execution of eT: 462.272 KB

  Calculation ended: 2021-01-27 19:44:45 UTC +01:00

  eT terminated successfully!
